core/src/main/scala/cromwell/core/path/RetryableFileSystemProviderProxy.scala

```diff
@@ -0,0 +1,51 @@
+package cromwell.core.path
+
+import java.net.URI
+import java.nio.channels.SeekableByteChannel
+import java.nio.file.DirectoryStream.Filter
+import java.nio.file._
+import java.nio.file.attribute.{BasicFileAttributes, FileAttribute, FileAttributeView}
+import java.nio.file.spi.FileSystemProvider
+import java.util
+
+import akka.actor.ActorSystem
+import cromwell.core.retry.Retry
+
+import scala.concurrent.{Await, Future}
+
+class RetryableFileSystemProviderProxy[T <: FileSystemProvider](delegate: T, retryParams: CustomRetryParams = CustomRetryParams.Default)(implicit actorSystem: ActorSystem) extends FileSystemProvider {
```

Comment:
I'm realizing now that this is not going to work - gonna have to find another way to do retries